docs/develop/tokenomics/dust-cnight/check-registration.mdx

```diff
@@ -0,0 +1,46 @@
+---
+SPDX-License-Identifier: Apache-2.0
+copyright: This file is part of midnight-docs. Copyright (C) 2025 Midnight Foundation. Licensed under the Apache License, Version 2.0 (the "License"); You may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+sidebar_label: "Check registration and UTxO status"
+---
+
+import { Step, StepsProvider } from '@site/src/components/Step';
+
+# Check registration and UTxO status
+
+Verify whether your Cardano wallet is registered for DUST, whether the mapped DUST address is valid, and which of your cNIGHT UTxOs are producing vs. not producing DUST.
+
+## Prerequisites
+
+- Cardano wallet address to inspect  
+- Access to a status view or tool that queries:
+  - the node’s public API (registration state and mapped DUST address validity)
+  - the observability/indexer (your cNIGHT UTxOs with creation/spent info)
+
+## Steps
+
+### Provide the wallet address to check
+<StepsProvider>
+  <Step>Select or enter the Cardano wallet address whose status you want to inspect.</Step>
+</StepsProvider>
+
+### Retrieve registration state and DUST address validity
+<StepsProvider>
+  <Step>Query the node public API for this wallet’s registration state.</Step>
+  <Step>Confirm whether a mapped DUST address exists and is recognized as valid by the node.</Step>
+  <Step>If not registered, no UTxOs are considered producing until registration is created and relayed.</Step>
+</StepsProvider>
+
+### List UTxOs and classify producing vs. non-producing
+<StepsProvider>
+  <Step>From the observability/indexer, obtain the wallet’s cNIGHT UTxOs (with creation time and spent status).</Step>
+  <Step>Determine the registration time (the time the registration UTxO was observed/accepted).</Step>
+  <Step>Producing UTxOs are cNIGHT UTxOs created at or after the registration time that remain unspent.</Step>
+  <Step>Non-producing UTxOs are cNIGHT UTxOs created before registration; activate them by re-spending to your own wallet.</Step>
```

Comment:
Note that they could be producing Dust for a different address if there was a different registration prior. Is that something we want to mention here?